lib/schema.js

```diff
@@ -77,23 +77,73 @@ const processSchema = (
   }
 };
 
+const getCategoryType = (category) => {
+  const type = category.constructor.name;
+  return type === 'Object' ? 'Local' : type;
```

Comment:
Do you mean this will return `Local` for all non-decorated categories? Need to be commented here and documented later.

Reply:
On the other hand, should we, maybe, wrap every undecorated category into `Local` on schema loading?